backend/minors/CGS.yaml

```diff
@@ -0,0 +1,132 @@
+---
+type: Minor
+name: Cognitive Science
+code: CGS
+degree:
+urls:
+- https://cogsci.princeton.edu/certificate
+description: Cognitive science is the interdisciplinary study of the 
+mind, brain, and intelligent systems, drawing predominantly on research 
+from psychology, philosophy, computer science, linguistics, neuroscience, 
+and anthropology. The interdisciplinary character of cognitive science is 
+reflected in its reliance on many levels of analysis and its employment of 
+a variety of methodologies appropriate to each level. The goal of cognitive 
+\science is to integrate the insights from multiple disciplines and modes of
+research into a unified scientific account of minds – be they human or 
+non-human, natural or artificial – and their place in nature and society.
+contacts:
+- type: Program Director
+  name: Anna Colasante
+  email: anna@princeton.edu
+req_list:
+  - name: Total Courses
+    explanation: "Students are required to take five courses in cognitive science"
+    min_needed: 5
+    max_counted: 1
+    req_list:
+    - name: Required Courses
+      explanation: At least one course designated with an asterisk in the list below, indicating that it is a CogSci Minor Required Course (CMRC);
+      min_needed: 1
+      max_counted: 1
```

Comment:
`max_counted` should be 5 since they can take more than 1 and have it count for the parent category